packages/mirage-flow-unix/mirage-flow-unix.1.4.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+maintainer:    "thomas@gazagnaire.org"
+homepage:      "https://github.com/mirage/mirage-flow"
+bug-reports:   "https://github.com/mirage/mirage-flow/issues"
+dev-repo:      "https://github.com/mirage/mirage-flow.git"
+doc:           "https://mirage.github.io/mirage-flow/"
+authors:       ["Thomas Gazagnaire" "Dave Scott"]
+tags:          [ "org:mirage"]
+license:       "ISC"
+
+build: [
+  [ "jbuilder" "subst" ] {pinned}
+  [ "jbuilder" "build" "-p" name "-j" jobs ]
+]
+
+build-test: [
+  [ "jbuilder" "runtest" ]
+]
+
+depends: [
+  "jbuilder"  {build & >="1.0+beta7"}
+  "fmt"
+  "logs"
+  "mirage-flow" {>= "1.2.0"}
+  "mirage-flow-lwt" {>= "1.2.0"}
+  "lwt"
+  "cstruct" {>= "2.3.0"}
```

Comment:
ok -- it wouldnt if there were other preprocessors in the mix but i guess it works iwth just cstruct_ppx